Redirects plugin: Add `is_redirect=true` data field to HTML redirect pages.

Garciat opened the pull request:
## Description

I want to hide redirect pages from my sitemap, and this small addition would help me like so:

```typescript
site.use(sitemap({ query: "is_redirect!=true" }));
```

I am not sure if there is a different preferred way to approach this. Please let me know 🙏 

## Related Issues

N/A

### Check List

- [x] Have you read the
      [CODE OF CONDUCT](https://github.com/lumeland/lume/blob/master/CODE_OF_CONDUCT.md)
- [x] Have you read the document
      [CONTRIBUTING](https://github.com/lumeland/lume/blob/master/CONTRIBUTING.md)
  - [x] One pull request per feature. If you want to do more than one thing,
        send multiple pull request.
  - [x] Write tests.
  - [x] Run deno `fmt` to fix the code format before commit.
  - [x] Document any change in the `CHANGELOG.md`.

oscarotero commented on plugins/redirects.ts:
```diff
@@ -123,7 +123,7 @@ function html(redirects: Redirect[], site: Site): void {
 </body>
 </html>`;
-    const page = Page.create({ url, content });
+    const page = Page.create({ url, content, is_redirect: true });
```
The variable should be camelCase: `isRedirect` or maybe `isRedirection`.

oscarotero reviewed (changes requested):
Good idea.
I just left a comment about the variable name. Maybe the sitemap should have this option enabled by default, what do you think?

Garciat commented:
> Maybe the sitemap should have this option enabled by default, what do you think?

I agree 👍 . I updated the default `query` for the sitemap plugin, and I added some tests to verify the change.
I'm not sure if that's the right way to document the default value.

Garciat commented:
Looking into the failed tests.

oscarotero commented:
Perfect. It will be available in the next Lume version!
